rewrite-test/src/main/kotlin/org/openrewrite/java/RemoveAnnotationTest.kt

```diff
@@ -0,0 +1,172 @@
+                    { a -> RemoveAnnotation.Scoped(a.classes[0], "lombok.RequiredArgsConstructor") },
+                    { a -> RemoveAnnotation.Scoped(a.classes[0].body.statements[0], "lombok.RequiredArgsConstructor") }
+            ),
+            before = """
+                package a;
+                
+                import lombok.RequiredArgsConstructor;
+                
+                @RequiredArgsConstructor
+                public class UsersController {
+                    @RequiredArgsConstructor
+                    class Inner {
+                    }
+                }
+            """,
+            after = """
+                package a;
+                
+                public class UsersController {
+                    class Inner {
+                    }
+                }
+            """
+    )
+
+    @Test
+    fun removeAnnotationFromField(jp: JavaParser) = assertRefactored(
+            jp,
+            visitorsMapped = listOf(
+                    { a -> RemoveAnnotation.Scoped((a as J.CompilationUnit).classes[0].fields[0], "b.MyAnnotation") },
+                    { a -> RemoveAnnotation.Scoped((a as J.CompilationUnit).classes[0].fields[1], "b.MyAnnotation") }
+            ),
+            before = """
+                package a;
+                
+                import b.MyAnnotation;
+                
+                public class UsersController {
+                    @MyAnnotation
+                    private final UserService userService;
+                
+                    @MyAnnotation
+                    NameService nameService;
+                }
+            """,
+            after = """
+                package a;
+                
+                public class UsersController {
+                    private final UserService userService;
+                    NameService nameService;
+                }
+            """
+    )
+
+    @Test
+    fun removeAnnotationFromMethod(jp: JavaParser) = assertRefactored(
+            jp,
+            dependencies = listOf(annot),
+            visitorsMappedToMany = listOf { a: J.CompilationUnit ->
+                a.classes[0].methods.map { m -> RemoveAnnotation.Scoped(m, "b.MyAnnotation") }
+            },
+            before = """
+                package a;
+                
+                import b.MyAnnotation;
+                
+                public class UsersController {
+                    @MyAnnotation
+                    UsersController() {
+                    }
+                
+                    @MyAnnotation
+                    public void onInit() {
+                    }
+                
+                    @MyAnnotation
+                    void onInit2() {
+                    }
+                    
+                    @MyAnnotation
+                    <T> T onInit3() {
+                        return null;
+                    }
+                }
+            """,
+            after = """
```

Comment:
You can align this further to the right. Just check that every blank line has the same indentation level.